graphics/mupdf/Portfile

```diff
@@ -58,13 +59,15 @@ build.args-append   CC=${configure.cc} \
                     HAVE_X11=no \
                     USE_SYSTEM_LIBS=yes \
                     build=release \
-                    verbose=yes
+                    verbose=yes \
+                    shared
```

Comment:
does making this change will require a revision bump for all the dependents of this port or not?

Reply:
Yes, there’s only one binary dependency that needed both a revision bump and a build fix. Done, thanks.

Reply:
can you please make that change to `zathura-plugin-pdf-mupdf` in a separate commit in this PR?

Reply:
Sure, updated.